### DISCUSSION TITLE
How to deploy NeMo-Guardrails in AWS Lambda function?

### DISCUSSION THREAD

**Discussion by mohilmakwana31** [NVIDIA-NeMo/Guardrails#642] [Q&A]
I am currently developing an chat application for a client and for security purposes I am using NeMo guardrails. But client wants to deploy NeMo guardrails in AWS lambda function which can be directly invoked by AWS Api gateway. 
So is there any way to deploy NeMo-Guardrails in AWS lambda function?

**Comment by manikandannp** [comment]
This needs to be answered. Most stuck with this
